tests/cxx/quoted_headers_first.cc

```diff
@@ -0,0 +1,36 @@
+//===--- prefix_header_operator_new.cc - test input file for iwyu ---------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// Tests that IWYU will respect the --quoted_headers_first option.
+
+#include <iostream>
+#include <list>
+#include <map>
+#include <memory>
+#include "subfolder/indirect_subfolder.h"
+
+std::unique_ptr<IndirectSubfolderClass> CreateIndirectSubfolderClass()
+{
```

Comment:
Opening brace goes on the preceding line

Reply:
Done.